Makefile

```diff
@@ -1,60 +1,141 @@
-# kernel-style V=1 build verbosity
-ifeq ("$(origin V)", "command line")
-       BUILD_VERBOSE = $(V)
+# Current Operator version
+VERSION ?= 2.0.0-snapshot
```

Comment:
This duplicates version information placed in `pkg/version/version.go`. Can we retrieve version information from there? Or at least cover update of this line in `bump-version.sh`?

Reply:
We use the version written in `pkg/version/version.go` to form the default image tag as well. The bump-version.sh script will update this line. [See](https://github.com/Kaitou786/kogito-cloud-operator/blob/KOGITO-3067/hack/bump-version.sh#L41)

Reply:
Thanks, I overlooked that